core/arch/arm/plat-ti/main.c

```diff
@@ -50,6 +50,7 @@
 static struct gic_data gic_data;
 static struct serial8250_uart_data console_data __early_bss;
 
+register_phys_mem(MEM_AREA_RAM_SEC, TZDRAM_BASE, CFG_TEE_RAM_VA_SIZE);
```

Comment:
This is needed, when building with CFG_WITH_PAGER the "tee-pageable.bin" is loaded to this area.